### DISCUSSION TITLE
Hjfatani

### DISCUSSION THREAD

**Discussion by Hjfatani** [community/community#61468] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.122.0 (104639223)
Device: iPhone15,3 (iOS 16.5.1)
</details>

**Comment by aconsuegra** [comment]
Hey @Hjfatani. Do you want to elaborate on the feedback you have about GitHub Mobile? Thanks in advance!
